submissions/kirill8210/oop-exercise/index.js

```diff
@@ -0,0 +1,54 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/kirill8210/a-tiny-JS-world
+   Web app: https://kirill8210.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+class Inhabitant{
+    constructor(species, name, gender, saying){
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.saying = saying;
+    }
+    getShow(){
+        return `${this.species}; ${this.name}; ${this.gender}; ${this.saying};`
+    }
+}
+
+class Human extends Inhabitant{
+    constructor(species, name, gender, saying, legs, hands) {
+        super(species, name, gender, saying);
+        this.legs = legs;
+        this.hands = hands;
+    }
+    getShow(){
+        return super.getShow() + ` ${this.legs}; ${this.hands};`
+    }
+}
+
+class Animals extends Inhabitant{
+    constructor(species, name, gender, saying, legs) {
+        super(species, name, gender, saying);
+        this.legs = legs;
```

Comment:
Make code DRYer by using OOP principles.
Why `legs` and e.g. `name` are treated in different ways?

Reply:
fixed legs to paws that only animals have